contrib/cmd/fs-idmap/fs-idmap.go

```diff
@@ -31,20 +31,22 @@ func main() {
 	if err := cmd.Start(); err != nil {
 		log.Fatalf("failed to run the helper binary: %v", err)
 	}
+	defer func() { _ = cmd.Wait() }()
```

Comment:
This is to make the linter happy. But I'll change it to check the return code and if it wasn't stopped by SIGKILL, then log it.

Reply:
You need to put `cmd.Process.Kill` here as well. Otherwise the process won't get killed by us in the error path.

Reply:
Right, added there. Thanks! :)